cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/model/ExecutableJar.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.packerina.model;
+
+import java.nio.file.Path;
+import java.util.HashSet;
+
+/**
+ * Class for holding contents of executable jar.
+ *
+ * @since 1.0.4
+ */
+public class ExecutableJar {
+
+    public Path moduleJar;
```

Comment:
Can this cause NPE in any case?

Reply:
module jar is the one create with actual ballerina source. Only place we are using is to get the init class name for main. So it  never can be null.